tweakwcs/tests/test_wcsimage.py

```diff
@@ -34,6 +34,16 @@ def rect_imcat(mock_fits_wcs):
     return w
 
 
+@pytest.fixture(scope='function')
+def rect_imcat_dup(mock_fits_wcs):
+    x = np.array([0.0, 0.0, 10.0, 10.0, 10.0, 0.0]) - 5
+    y = np.array([0.0, 10.0, 10.0, 0.0, 0.0, 0.0]) - 5
```

Comment:
I realized this (set of **input** data) actually does not produce a convex hull with two very close (or identical) vertices.